index.html

```diff
@@ -2608,8 +2608,11 @@ <h3>WoT Security and Privacy Guidelines</h3>
     <section id="sec-servient-implementation" class="informative">
         <h1>Servient Implementation</h1>
         <p>
-            A <a>Servient</a> is a software stack that implements the WoT building blocks presented in the previous section.
-            <a>Servients</a> can host and expose <a>Things</a> and/or consume <a>Things</a> (i.e., host <a>Consumers</a>).
+            As described in section <a href="sec-WoT-servient-architecture-high-leve"></a> 
```

Comment:
This might be: `<a href="#sec-WoT-servient-architecture-high-level"></a>`

Reply:
Should also be all lower case (cf. WoT)